net/shorewall-core/Makefile

```diff
@@ -0,0 +1,58 @@
+#
+# Copyright (C) 2008-2017 OpenWrt.org
```

Comment:
https://forum.lede-project.org/t/packaging-guidelines/65/2
Please read "Copyright statements in Makefiles"